tests/integrations/gcp/test_gcp.py

```diff
@@ -0,0 +1,384 @@
+"""
+# GCP Cloud Functions system tests
+
+"""
+import json
+import time
+from textwrap import dedent
+import uuid
+import tempfile
+import shutil
+import sys
+import subprocess
+
+import pytest
+import os.path
+import os
+
+requests = pytest.importorskip("requests")
```

Comment:
I would importorskip the google cloud sdk additionally

Reply:
Sure I'll do that as well.